### PR TITLE
feature/MBA-86_create-all-transactions-screen

### PR DESCRIPTION
**Pull request by milkanuch**

### CODE REVIEW

**Review by stepan103ua — APPROVED** [review]
LGTM

File: src/screens/AllTransactionsScreen/AllTransactionsItem/AlllTransactionsItem.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { colors } from '../../../constants/colors';
+
+import {
+  IAllTransactionsItemProps,
+  TransactionsCategory,
+} from './allTransactionItem.types';
+import styles from './allTransactionsItem.styles';
+import { activeOpacityValue, IconSize } from './allTransactionItem.settings';
+
+const AlllTransactionsItem: FC<IAllTransactionsItemProps> = ({
```
Comment (VadymBatryn) [inline src/screens/AllTransactionsScreen/AllTransactionsItem/AlllTransactionsItem.tsx:15]: Fix the typo "Alll" and the name of the file also.

File: src/screens/AllTransactionsScreen/AllTransactionsItem/AlllTransactionsItem.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <Icon name={category} size={IconSize} color={colors.black} />
+        <Text style={styles.cardProvider}>{provider}</Text>
+      </View>
+      <View>
+        <Text style={styles.transactionSum}>
+          {profit ? `+${sum.toFixed(0)}` : (-sum).toFixed(0)}
+          {` ${currency}`}
+        </Text>
+        <Text style={styles.transactionDate}>
+          {new Date(time).toLocaleDateString()}
```
Comment (VadymBatryn) [inline src/screens/AllTransactionsScreen/AllTransactionsItem/AlllTransactionsItem.tsx:40]: Will this variable recreate each rerender?

File: src/screens/AllTransactionsScreen/AllTransactionsItem/AlllTransactionsItem.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      activeOpacity={activeOpacityValue}
+      onPress={handleTransactionFullDescription}>
+      <View style={styles.categoryContainer}>
+        <Icon name={category} size={IconSize} color={colors.black} />
+        <Text style={styles.cardProvider}>{provider}</Text>
+      </View>
+      <View>
+        <Text style={styles.transactionSum}>
+          {profit ? `+${sum.toFixed(0)}` : (-sum).toFixed(0)}
+          {` ${currency}`}
```
Comment (VadymBatryn) [inline src/screens/AllTransactionsScreen/AllTransactionsItem/AlllTransactionsItem.tsx:37]: Looks too complex, try to shorten it.

You can try something like that `${(profit ? 1 : -1) * sum.toFixed(0) + currency}` or create another version

**Review by VadymBatryn — CHANGES_REQUESTED** [review]

File: src/screens/AllTransactionsScreen/AllTransactionsItem/AlllTransactionsItem.tsx (hunk 2 above)
Comment (milkanuch) [reply src/screens/AllTransactionsScreen/AllTransactionsItem/AlllTransactionsItem.tsx:40]: No!

**Review by JeriRov — APPROVED** [review]
LGTM

**Review by VadymBatryn — APPROVED** [review]

**Pull request merged**